content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions.md

```diff
@@ -347,57 +684,116 @@ kubectl apply -f my-crontab.yaml
 crontab "my-new-cron-object" created
 ```
 
-### Publish Validation Schema in OpenAPI v2
+### Defaulting
 
-{{< feature-state state="alpha" for_kubernetes_version="1.14" >}}
+{{< feature-state state="alpha" for_kubernetes_version="1.15" >}}
 
-Starting with Kubernetes 1.14, [custom resource validation schema](#validation) can be published as part
-of [OpenAPI v2 spec](/docs/concepts/overview/kubernetes-api/#openapi-and-swagger-definitions) from
-Kubernetes API server.
+{{< note >}}
+Defaulting is available as alpha since 1.15. The
+`CustomResourceDefaulting` feature must be enabled, which is the case automatically for many clusters for beta features. Please refer to the [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) documentation for more information.
```

Comment:
strike "which is the case automatically for many clusters for beta features" until this is a beta feature

Reply:
fixed